### PR TITLE
Code formatting and fix yuidoc #7

### PR DESCRIPTION
**Pull request by hvostya**
TFS#105815.

### CODE REVIEW

File: addon/components/flexberry-groupedit.js [hunk 1]
```diff
@@ -1,317 +1,416 @@
 /**
- * @module ember-flexberry
- */
+ @module ember-flexberry
```
Comment (yzaripova) [inline addon/components/flexberry-groupedit.js]: One space or two?

File: addon/components/flexberry-groupedit.js [hunk 2]
```diff
@@ -1,317 +1,416 @@
...
 /**
- * Component for create, edit and delete detail objects.
- *
- * @class FlexberryGroupedit
- * @extends FlexberryBaseComponent
- */
+  Component for create, edit and delete detail objects.
+
+  On order to add component following template can be used:
+  ```handlebars
```
Comment (yzaripova) [inline addon/components/flexberry-groupedit.js]: Add please @example.

File: addon/components/flexberry-groupedit.js [hunk 3]
```diff
@@ -1,317 +1,416 @@
...
+    componentProperties: null
   },
 
   /**
-    Text to be displayed in table body, if content is not defined or empty.
+    Content to be displayed (models collection).
 
-    @property placeholder
+    @property content
+    @type ManyArray
```
Comment (yzaripova) [inline addon/components/flexberry-groupedit.js]: Replace please DS.ManyArray.

File: addon/components/flexberry-groupedit.js [hunk 4]
```diff
@@ -1,317 +1,416 @@
...
-   * @property allowColumnResize
-   * @type Boolean
-   * @default true
-   */
-  allowColumnResize: true,
+    Flag: indicates whether to show delete button at toolbar.
+
+    @property deleteButton
+    @type Boolean
+    @default false
```
Comment (yzaripova) [inline addon/components/flexberry-groupedit.js]: @default if false, but realy is true?

File: addon/components/flexberry-groupedit.js [hunk 5]
```diff
@@ -1,317 +1,416 @@
...
-   *
-   * @property singleColumnHeaderTitle
-   * @type String
-   */
-  singleColumnHeaderTitle: undefined,
+   Text to be displayed in table body, if content is not defined or empty.
+
+   @property placeholder
+   @type String
+   @default 't('components.flexberry-groupedit.placeholder')'
```
Comment (yzaripova) [inline addon/components/flexberry-groupedit.js]: Now I write this without quotes :)

File: addon/components/flexberry-groupedit.js [hunk 6]
```diff
@@ -1,317 +1,416 @@
...
+     { {flexberry-groupedit
+       ...
+       sorting=computedSorting)} }
+    ```
+
+    where {{#crossLink "SortableControllerMixin/computedSorting:property"}}computedSorting is property of SortableControllerMixin{{/crossLink}}.
+
+    @property sorting
+    @type Object
+  */
```
Comment (yzaripova) [inline addon/components/flexberry-groupedit.js]: @default null?

File: addon/components/flexberry-groupedit.js [hunk 7]
```diff
@@ -1,317 +1,416 @@
...
 
   /**
-   * Custom classes for table.
-   *
-   * @property customTableClass
-   * @type String
-   * @default ''
-   */
-  customTableClass: '',
+     Default cell component that will be used to display values in single column.
```
Comment (yzaripova) [inline addon/components/flexberry-groupedit.js]: three spaces?

File: addon/components/flexberry-groupedit.js [hunk 8]
```diff
@@ -1,317 +1,416 @@
...
 
   /**
-   * Flag: indicates whether to show creation button at toolbar.
-   *
-   * @property createNewButton
-   * @type Boolean
-   * @default false
-   */
-  createNewButton: true,
+     Header title of single column.
```
Comment (yzaripova) [inline addon/components/flexberry-groupedit.js]: three spaces?

File: addon/components/flexberry-groupedit.js [hunk 9]
```diff
@@ -1,317 +1,416 @@
...
-   */
-  saveBeforeRouteLeave: false
+  actions: {
+    /**
+      Handles action from object-list-view when no handler for this component is defined.
+
+      @example
+      If sorting is used then there has to be declaration:
+      ```
+      {{flexberry-groupedit
```
Comment (yzaripova) [inline addon/components/flexberry-groupedit.js]: May be replace this code, like this:
{{...
  // two spaces
}}

File: addon/components/flexberry-groupedit.js [hunk 10]
```diff
@@ -1,317 +1,416 @@
...
+                      'Set handler like {{flexberry-groupedit ... sortByColumn=(action "sortByColumn")}}.');
+    },
+
+    /**
+      Handles action from object-list-view when no handler for this component is defined.
+
+      @example
+      If sorting is used then there has to be declaration:
+      ```
+      {{flexberry-groupedit
```
Comment (yzaripova) [inline addon/components/flexberry-groupedit.js]: May be replace this code, like this:
{{...
// two spaces
}}

File: addon/components/flexberry-groupedit.js [hunk 11]
```diff
@@ -1,317 +1,416 @@
...
+       ...
+       orderable=true
+       sorting=computedSorting
+       sortByColumn=(action "sortByColumn")
+       addColumnToSorting=(action "addColumnToSorting")}}
+      ```
+
+      @method actions.addColumnToSorting
+      @param {Object} column Column to add sorting by.
```
Comment (yzaripova) [inline addon/components/flexberry-groupedit.js]: I think the point here is not necessary.
See us task:
  /**
    Query for records that meet certain criteria. Resolves with [DS.RecordArray](http://emberjs.com/api/data/classes/DS.RecordArray.html).
    For more information see [query](http://emberjs.com/api/data/classes/DS.Store.html#method_query) method of [DS.Store](http://emberjs.com/api/data/classes/DS.Store.html).
    @method query
    @param {String} modelName
    @param {Object} query
    @param {Boolean} [query.useOnlineStore] Allow to explicitly specify online or offline store using independently of global online status
    @return {Promise} promise
  */

File: addon/components/groupedit-toolbar.js [hunk 12]
```diff
@@ -1,148 +1,162 @@
...
 
   /**
-   * The collection of functions that will be invoked when
-   * click on toolbar buttons.
-   *
-   * @property actions
-   * @type Object
-   * @readOnly
-   */
+    The collection of functions that will be invoked when
```
Comment (yzaripova) [inline addon/components/groupedit-toolbar.js]: Is it really necessary?

File: addon/components/groupedit-toolbar.js [hunk 13]
```diff
@@ -1,148 +1,162 @@
...
+      {{#crossLink "FlexberryGroupeditComponent"}}{{/crossLink}}.
+
+      @method actions.addRow
+    */
+    addRow() {
       if (this.get('readonly')) {
         return;
       }
 
       var componentName = this.get('componentName');
```
Comment (yzaripova) [inline addon/components/groupedit-toolbar.js]: Replace please to let.

File: addon/components/groupedit-toolbar.js [hunk 14]
```diff
@@ -1,148 +1,162 @@
...
+      {{#crossLink "FlexberryGroupeditComponent"}}{{/crossLink}}.
+
+      @method actions.deleteRows
+    */
+    deleteRows() {
       if (this.get('readonly')) {
         return;
       }
 
       var componentName = this.get('componentName');
```
Comment (yzaripova) [inline addon/components/groupedit-toolbar.js]: Replace please to let.

File: addon/components/groupedit-toolbar.js [hunk 15]
```diff
@@ -1,148 +1,162 @@
...
-      this.get('groupEditEventsService').deleteRowsTrigger(componentName);
+      this.get('_groupEditEventsService').deleteRowsTrigger(componentName);
+    }
+  },
+
+  /**
+    The method called when component is instantiated.
```
Comment (yzaripova) [inline addon/components/groupedit-toolbar.js]: It is ember's method.
I wrile like this:
"/**
     An overridable method called when objects are instantiated.
     For more information see [init](http://emberjs.com/api/classes/Ember.View.html#method_init) method of [Ember.View](http://emberjs.com/api/classes/Ember.View.html).
*/"

File: addon/components/flexberry-groupedit.js (hunk 1 above)
Comment (hvostya) [inline addon/components/flexberry-groupedit.js]: Fixed.

File: addon/components/groupedit-toolbar.js [hunk 16]
```diff
@@ -1,148 +1,162 @@
...
   },
 
   /**
-   * Boolean flag to indicate enabled state of delete rows button.
-   *
-   * @property isDeleteRowsEnabled
-   * @type Boolean
-   */
-  isDeleteRowsEnabled: undefined,
+    Implementation of component's teardown.
```
Comment (yzaripova) [inline addon/components/groupedit-toolbar.js]: Override to implement teardown.
For more information see [willDestroy](http://emberjs.com/api/classes/Ember.Component.html#method_willDestroy) method of [Ember.Component](http://emberjs.com/api/classes/Ember.Component.html).

File: addon/components/mobile/flexberry-groupedit.js [hunk 17]
```diff
@@ -1,91 +1,131 @@
...
-   * @default false
-   */
-  useSingleColumn: true,
+    Flag: indicates whether ordering by clicking on column headers is allowed.
+
+    It overrides default value of
+    {{#crossLink "FlexberryGroupeditComponent/orderable:property"}}{{/crossLink}}
+    of base component.
+
+    @property headerClickable
```
Comment (yzaripova) [inline addon/components/mobile/flexberry-groupedit.js]: @property headerClickable, but really is orderable

File: addon/mixins/flexberry-groupedit-route.js [hunk 18]
```diff
@@ -1,64 +1,50 @@
...
-  },
-
-  deactivate() {
-    this._super(...arguments);
-
-    this.get('groupEditEventsService').off('olvRowAdded', this, this._rowAdded);
-    this.get('groupEditEventsService').off('olvRowDeleted', this, this._rowDeleted);
-    this.get('groupEditEventsService').off('olvRowsChanged', this, this._rowChanged);
-  },
+    @private
```
Comment (yzaripova) [inline addon/mixins/flexberry-groupedit-route.js]: @property should be the first.

File: addon/mixins/flexberry-groupedit-route.js [hunk 19]
```diff
@@ -126,42 +112,83 @@ export default Ember.Mixin.create({
...
+  },
+
+  /**
+    Event handler for "row has been selected" event in {{#crossLink "FlexberryGroupeditComponent"}}{{/crossLink}}.
+
+    @method _rowAdded
+    @private
+
+    @param {String} componentName The name of {{#crossLink "FlexberryGroupeditComponent"}}{{/crossLink}}.
+    @param {Model} record The model corresponding to added row in {{#crossLink "FlexberryGroupeditComponent"}}{{/crossLink}}.
```
Comment (yzaripova) [inline addon/mixins/flexberry-groupedit-route.js]: DS.Model?

File: addon/components/flexberry-groupedit.js (hunk 11 above)
Comment (hvostya) [inline addon/components/flexberry-groupedit.js]: To my mind a dot has to be set at the end of the parameter comment. We use uppercase letter at the beginning so we have to set a dot.

File: addon/models/new-platform-flexberry-flexberry-user-setting.js [hunk 20]
```diff
@@ -1,11 +1,17 @@
...
 import BaseModel from './base';
 import Proj from 'ember-flexberry-data';
 
+/**
+  Model to work with user settings .
+
+  @class NewPlatformFlexberryFlexberryUserSettingModel
+  @extends BaseModel
+*/
 var Model = BaseModel.extend({
```
Comment (yzaripova) [inline addon/models/new-platform-flexberry-flexberry-user-setting.js]: Replace please to let.

File: addon/serializers/new-platform-flexberry-flexberry-user-setting.js [hunk 21]
```diff
@@ -1,8 +1,14 @@
 /**
- * @module ember-flexberry
- */
+  @module ember-flexberry
+*/
 import ODataSerializer from 'ember-flexberry-data/serializers/odata';
 
+/**
+  Serializer .
```
Comment (yzaripova) [inline addon/serializers/new-platform-flexberry-flexberry-user-setting.js]: Space?

File: addon/services/user-settings-service.js [hunk 22]
```diff
@@ -1,53 +1,82 @@
...
-   * @return {Promise} A promise. It returns saving result
-   */
-  saveUserSetting: function(options) {
+   It deletes user setting from server by setting's and module's names.
+
+   @method deleteUserSetting
+
+   @param {Object} [options] Parameters for user setting getting.
+   @param {String} options.moduleName Name of module to search by.
+   @param {String} options.settingName Setting name to search by..
```
Comment (yzaripova) [inline addon/services/user-settings-service.js]: Two dot.

**Pull request merged**